topology-graph.js

```diff
@@ -28,7 +28,7 @@
     /* A cache to prevent jumping when rapidly toggling views */
     var cache = { };
 
-    function topology_graph(selector, force, notify) {
+    function topology_graph(selector, force, notify, customRadius) {
```

Comment:
> Could we make this an 'options' argument so it's more extensible. so there would be:
> 
> topology_graph(selector, notify, options)
> 
> And options would have:
> 
> force
> radius
> ...
> What do you think?

I don't oppose to the concept, however moving force to be inside options will break existing users